cookbooks/bach_krb5/recipes/gems.rb

```diff
@@ -0,0 +1,32 @@
+require 'pathname'
+require 'rubygems'
+gem_path = Pathname.new(Gem.ruby).dirname.join("gem").to_s
+
+gem_package "rake-compiler" do
+  gem_binary gem_path
+  version ">=0.0.0"
+  action :nothing
+end.run_action(:install)
+
+node['krb5']['devel']['packages'].each do |pkg|
+  package pkg do
+    action :nothing
+  end.run_action(:install)
+end
+
+gem_package "rkerberos" do
+  gem_binary gem_path
+  version ">=0.0.0"
+  action :nothing
+end.run_action(:install)
+
+execute "correct-permissions-for-rkerberos" do
+  command "chmod 644 /opt/chef/embedded/lib/ruby/gems/1.9.1/specifications/rkerberos-0.1.3.gemspec"
```

Comment:
Can not this be a [file](https://docs.chef.io/resource_file.html) resource with a `:touch` action?